.github/workflows/publish_dockerhub_images.yml

```diff
@@ -0,0 +1,95 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+name: Build And Publish DockerHub images
+on:
+  workflow_dispatch:
+    inputs:
+      RELEASE_VERSION:
+        type: string
+        description: Release Version
+        required: true
+        default: "2.42.X"
+      RC_NUM:
+        type: string
+        description: Release Candidate Number
+        required: true
+        default: "1"
+      FINAL_PUBLISH:
+        type: boolean
+        description: Publish as RELEASE_VERSION and LATEST?
+        required: true
+        default: false
+
+jobs:
+  build:
+    env:
+      RELEASE_VERSION: ${{github.event.inputs.RELEASE_VERSION}}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{secrets.DOCKER_HUB_REPO}}
+          password: ${{secrets.DOCKER_HUB_TOKEN}}
+      - name: Setting Release Branch
+        run: echo "RC_TAG=v${{env.RELEASE_VERSION}}-RC${{github.event.inputs.RC_NUM}}" >> $GITHUB_ENV    
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{env.RC_TAG}}
+      - name: Setup Go 
+        uses: actions/setup-go@v3
+        with:
+          go-version: '^1.13.1' 
+      - name: Setup python 
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+        
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with: 
+          distribution: 'temurin'
+          java-version: |
+            8
+            17
+            11
```

Comment:
Do we actually need all 3 versions of Java? Is there a reason they're ordered like this (not 8, 11, 17)?

Reply:
According to the release instructions we need at least Java 8 and 11, the reason behind this sequence is that the default version installed should be 11 (the last one specified is taken as default), Java 8 path is required as reference in the gradlew task, I will check if Java 17 can be removed.

Reply:
Java 17 was not required, I'm leaving versions 8 and 11, and Go Version 1.19.